lxd/instance/drivers/driver_qemu_templates.go

```diff
@@ -358,6 +358,28 @@ multifunction = "on"
 {{- end }}
 `))
 
+var qemuVirtioFSDriveConfig = template.Must(template.New("qemuVirtioFSDriveConfig").Parse(`
+# Config drive (using virtio-fs)
+[chardev "qemu_config"]
+backend = "socket"
+path = "{{.path}}"
+
+[device "dev-qemu_config-virtiofs"]
+{{- if eq .bus "pci" "pcie"}}
+driver = "vhost-user-fs-pci"
+bus = "{{.devBus}}"
+addr = "{{.devAddr}}"
+{{- end}}
+{{if eq .bus "ccw" -}}
+driver = "vhost-user-fs-ccw"
+{{- end}}
+chardev = "qemu_config"
+tag = "config-virtiofs"
```

Comment:
Is that the mount_tag equivalent? If so, any reason it can't be `config`?
It shouldn't be conflicting with 9p since they're two different protocols.